weights.json

```diff
@@ -40,6 +40,19 @@
       "N\u0000sourcecred\u0000discourse\u0000topic\u0000https://discourse.sourcecred.io\u0000270\u0000": 4,
       "N\u0000sourcecred\u0000discourse\u0000topic\u0000https://discourse.sourcecred.io\u0000291\u0000": 4,
       "N\u0000sourcecred\u0000discourse\u0000topic\u0000https://discourse.sourcecred.io\u0000327\u0000": 4,
+      "N\u0000sourcecred\u0000github\u0000PULL\u0000sourcecred\u0000sourcecred\u00001544\u0000": 0,
```

Comment:
The spam in question consisted of issues, but these addresses look like
the addresses of pull requests. Should each `PULL` be `ISSUE`? Did you
get these addresses from some graph or enter them in manually?

Reply:
Thanks for catching this--I made this change hastily and manually. Will fix.